MiniShopify/src/main/java/springboot/model/Cart.java

```diff
@@ -0,0 +1,51 @@
+package springboot.model;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+public class Cart {
+
+    @Id
+    @GeneratedValue
+    private long id;
+
+    @OneToOne
+    private User owner;
+
```

Comment:
I think you need an empty constructor for all entities

Reply:
I just wanted to make sure the relationships are correct. Im doing those rn though